### PR TITLE
Add e2e test action for cypress plugin

### PR DESCRIPTION
**Pull request by ryanjduffy**
Extends the earthly script to run the [flake](https://github.com/replayio-public/flake) benchmark repo with the current (unreleased) cypress plugin code to identify regressions.

Next will be to extend this with metadata test cases from the produced replays but for now we can verify with the replay links.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -7,7 +7,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
```
Comment (ryanjduffy) [inline .github/workflows/ci.yml:10]: This should probably be in the earthfile via `GIT CLONE` but I started with this so it's here for now.

Comment (ryanjduffy) [reply .github/workflows/ci.yml:10]: Removing this means we can remove the ignore file changes too

File: Earthfile [hunk 2]
```diff
@@ -15,6 +16,38 @@ test:
   FROM +build
   RUN npm test
 
+setup:
+  FROM +build
+  RUN apt update && apt install xz-utils
+  RUN npx @replayio/playwright install
+  # download binary openssl packages from Impish builds
+  RUN wget http://security.ubuntu.com/ubuntu/pool/main/o/openssl/openssl_1.1.1f-1ubuntu2.19_amd64.deb
```
Comment (ryanjduffy) [inline Earthfile:24]: Adds support for openssl 1.1.1

File: package.json [hunk 3]
```diff
@@ -17,7 +17,8 @@
   "license": "ISC",
   "devDependencies": {
     "lerna": "^4.0.0",
-    "prettier": "^2.7.1"
+    "prettier": "^2.7.1",
+    "typescript": "^5.1.6"
```
Comment (ryanjduffy) [inline package.json:21]: The build steps for the packages were failing because `tsc` wasn't installed.

File: Earthfile [hunk 4]
```diff
@@ -1,11 +1,12 @@
-VERSION 0.6
-FROM node:lts-alpine
+VERSION --wait-block 0.6
```
Comment (jazzdan) [inline Earthfile:1]: Do we need `--wait-block`?

**Review by jazzdan — APPROVED** [review]

File: Earthfile (hunk 4 above)
Comment (ryanjduffy) [reply Earthfile:1]: Nope. I tried to use `WAIT` at one point but we can drop this now.

**Pull request merged**